test/test_log.py

```diff
@@ -156,6 +157,15 @@ def test_long_string_logging(self, backend: FakeModelBackend, logger: logging.Lo
         assert len(calls[1][1]) == MAX_LOG_LINE_LEN
         assert len(calls[2][1]) == 9
 
+    def test_warning(self, backend: FakeModelBackend):
+        ops.log.setup_root_logging(backend)
+        warnings.warn('warning')
```

Comment:
Probably best to use some other string, in case we had a bug that printed out the log level instead of the message.